src/pages/iou/steps/NewRequestAmountPage.js

```diff
@@ -59,6 +62,7 @@ function NewRequestAmountPage({route, iou, report}) {
     const reportID = lodashGet(route, 'params.reportID', '');
     const isEditing = lodashGet(route, 'path', '').includes('amount');
     const currentCurrency = lodashGet(route, 'params.currency', '');
+    const isDistanceRequest = selectedTab === CONST.TAB.DISTANCE;
```

Comment:
```suggestion
    const isDistanceRequestTab = selectedTab === CONST.TAB.DISTANCE;
```

Reply:
I prefer keeping the original name. This variable rather represents if the money request that is being drafted is a distance request or not (not necessarily associated with tab selection).

NewRequestAmountPage is used in the following two scenarios (the left one is a full screen without a tab):

<img width="200" alt="Screenshot 2023-08-30 at 11 21 00 AM" src="https://github.com/Expensify/App/assets/98560306/b7ccd628-76dc-453d-b870-e05991d6effa">
<img width="200" alt="Screenshot 2023-08-30 at 11 20 31 AM" src="https://github.com/Expensify/App/assets/98560306/cc3bf875-f876-4074-8eab-2121251eb36b">

I decided to use the latest tab selection as a way to check if the money request is a distance request or not. We discussed if we should refactor the URL path so that we can get this information from the URL path.

However, I decided to postpone this because I think we should not support deep linking to screens for different stages of the money request flow to begin with 🤔 

So for now, I temporarily use the tab selection information to determine the kind of money request being drafted.